### PR TITLE
First danish translation 

### PR DESCRIPTION
**Pull request by LPSGizmo**

### CODE REVIEW

File: data/language/da-DK.txt [hunk 1]
```diff
@@ -1040,120 +1040,120 @@ STR_1034    :Kan kun bygges på jorden!
...
-STR_1133    :Vehicle {POP16}{COMMA16}
+STR_1088    :Intens tilstand
+STR_1089    :Berserk tilstand
+STR_1090    :Spøgelseshus tilstand
+STR_1091    :Cirkus tilstand
+STR_1092    :Nedadgående afsendelse
+STR_1093    :Hurlumhejhus tilstand
+STR_1094    :Frit-fald tilstand
+STR_1095    :Kontinuerlig kredsløb, blok sektion tilstand
+STR_1096    :Kraftfuld afsendelse (without passing station)
```
Comment (Gymnasiast) [inline data/language/da-DK.txt]: You forgot the second half of this string.

File: data/language/da-DK.txt [hunk 2]
```diff
@@ -2156,74 +2156,74 @@ STR_2148    :Varme chokolader
...
-STR_2179    :some Soybean Milk
-STR_2180    :some Sujeonggwa
-STR_2181    :a Sub Sandwich
+STR_2160    :Småkager
+STR_2161    :Tomme skåle
+STR_2162    :Tomme Drikkekartoner
+STR_2163    :Tomme Juiceglas 
+STR_2164    :Ristede pølser
+STR_2165    :Tomme skåle
+STR_2166    :et On-Ride Foto
```
[the remark this answers is not in the document]
Comment (LPSGizmo) [reply data/language/da-DK.txt]: Not that i know of. Tried to ask in "Djurs Sommerland", an amusement park here in Denmark.
Everyone just called them a "foto"
But i have purchased the original RCT3 (Atari) witch is translated to Danish (by Atari). So I will get back to this, when I have installed the game.

**Comment by Gymnasiast** [comment]
You did good (and fast) work so far. I hope you can find the time to do the rest soon.

**Comment by LPSGizmo** [comment]
Up to STR_5814 (up to tooltips part) is now done.
Do I need to make a new pull request?

Sendt fra Mail<https://go.microsoft.com/fwlink/?LinkId=550986> til Windows 10

________________________________
Fra: Michael Steenbeek <notifications@github.com>
Sendt: Friday, July 27, 2018 12:15:48 PM
Til: OpenRCT2/Localisation
Cc: Lars Sørensen; Author
Emne: Re: [OpenRCT2/Localisation] First danish translation (#1515)


You did good (and fast) work so far. I hope you can find the time to do the rest soon.

**Comment by Gymnasiast** [comment]
Yes, indeed.

**Comment by LPSGizmo** [comment]
[cid:image003.png@01D42EFC.BE90BD60]
Sendt fra Mail<https://go.microsoft.com/fwlink/?LinkId=550986> til Windows 10

________________________________
Fra: Michael Steenbeek <notifications@github.com>
Sendt: Tuesday, August 7, 2018 5:47:24 PM
Til: OpenRCT2/Localisation
Cc: Lars Sørensen; Author
Emne: Re: [OpenRCT2/Localisation] First danish translation (#1515)


Yes, indeed.

**Pull request merged**